src/main/java/org/example/Services/CustomerServices.java

```diff
@@ -27,27 +28,54 @@ public void depositMoney(String customerName, int depositAmount) throws IOExcept
         }
     }
 
+    public void makeTransfer(String nameSenderCustomer, String nameReceiverCustomer, int amountToSend) throws SQLException, IOException {
+        int senderreturnedId = dbcontroller.findCustomerByUsername(nameSenderCustomer);
+        int receiverreturnedId = dbcontroller.findCustomerByUsername(nameReceiverCustomer);
 
-    public boolean sendMoneyToSomeone(String nameSenderCustomer, String nameReceiverCustomer, int amountToSend) throws IOException, SQLException {
+        boolean sendmoney = sendMoneyToSomeone(senderreturnedId, amountToSend);
+        if (sendmoney){

```

Comment:
Since it's a bool I assume it checks whether the transaction was successful... so I suggest changing the variable to reflect that:
```suggestion
        boolean isTransactionSuccessful = sendMoneyToSomeone(senderreturnedId, amountToSend);
        if (isTransactionSuccessful){
```
As it is the variable `sendmoney` is hard to understand just by reading it in isolation.